lib/core/src/Cardano/Pool/Ranking.hs

```diff
@@ -181,6 +183,16 @@ data EpochConstants = EpochConstants
       -- ^ Total rewards in an epoch. "R" in the spec.
     } deriving (Show, Eq, Generic)
 
+instance Buildable EpochConstants where
+    build ec = blockListF' "" id
+        [ "leaderStakeInfluence (a0) =  "
+            <> build (getNonNegative $ leaderStakeInfluence ec)
+        , "desiredNumberOfPool (k) = "
+            <> build (getPositive $ desiredNumberOfPools ec)
+        , "totalRewards (R) = "
+            <> build (getQuantity $ totalRewards ec)
+        ]
```

Comment:
Running integration tests (with temporary tweaked R):
```
[cardano-wallet.pools-engine:Info:114] [2020-01-22 11:46:29.32 UTC] Downloading http://localhost:49177/test-integration-registry.zip -> /private/var/folders/h_/mlzym27d5lz_fx3ph2d547700000gn/T/stake-pool-metadata-564df08caee116ef/registry.zip: Using cached file last modified at 2020-01-22 11:45:27.974117167 UTC
[cardano-wallet.api-server:Info:114] [2020-01-22 11:46:29.32 UTC] [RequestId 601] 200 OK in 0.007592s
[cardano-wallet.api-server:Info:114] [2020-01-22 11:46:29.82 UTC] [RequestId 602] [GET] /v2/stake-pools
[cardano-wallet.pools-engine:Info:114] [2020-01-22 11:46:29.83 UTC] Using ranking epoch constants:  leaderStakeInfluence (a0) =  0.0
 desiredNumberOfPool (k) = 10
 totalRewards (R) = 10000

[cardano-wallet.pools-engine:Info:114] [2020-01-22 11:46:29.83 UTC] Downloading http://localhost:49177/test-integration-registry.zip -> /private/var/folders/h_/mlzym27d5lz_fx3ph2d547700000gn/T/stake-pool-metadata-564df08caee116ef/registry.zip: Using cached file last modified at 2020-01-22 11:45:27.974117167 UTC
[cardano-wallet.api-server:Info:114] [2020-01-22 11:46:29.83 UTC] [RequestId 602] 200 OK in 0.008747s
[cardano-wallet.api-server:Info:114] [2020-01-22 11:46:30.33 UTC] [RequestId 603] [GET] /v2/stake-pools
[cardano-wallet.pools-engine:Info:114] [2020-01-22 11:46:30.34 UTC] Using ranking epoch constants:  leaderStakeInfluence (a0) =  0.0
 desiredNumberOfPool (k) = 10
 totalRewards (R) = 10000
```

Reply:
This would look even better with an extra newline before the leaderStakeInfluence, don't you think ?